apps/src/block_utils.js

```diff
@@ -510,9 +525,13 @@ exports.createJsWrapperBlockCreator = function (
           });
         }
 
-        interpolateInputs(blockly, this, determineInputs(blockText, inputs));
+        if (inline === false) {
+          addInputs(blockly, this, args);
+        } else {
+          interpolateInputs(blockly, this, determineInputs(blockText, inputs));
+          this.setInputsInline(true);
+        }
```

Comment:
this changes the default behavior; will that break anything?

Reply:
For existing blocks that don't set the `inline` option, this is supposed to be exactly the same as before. How is it changing?

Reply:
Oh, I misread this!

I don't super love the two different expected values for the `inline` flag be `false` and `undefined`; falsiness tends to be fragile. But it does make sense for this application, so I'll say LGTM